libyul/AsmParser.cpp

```diff
@@ -200,13 +200,13 @@ Statement Parser::parseStatement()
 
 	if (holds_alternative<Identifier>(elementary))
 	{
-		Identifier& identifier = std::get<Identifier>(elementary);
+		Identifier identifier = std::get<Identifier>(move(elementary));
 		return ExpressionStatement{identifier.location, { move(identifier) }};
```

Comment:
Not sure what is the best approach on these two lines regarding move semantics, but ref + move seems to be weird.